.gitignore

```diff
@@ -0,0 +1,5 @@
+images

```

Comment:
Images and screenshots are to be added to version control and should not be added here